### PR TITLE
implemented sbi testcases for sbi spec section 3.5

### PR DESCRIPTION
**Pull request by 1danish159**
# Pull Request Template

## Prequisites

- [x] Did you run lint -cf `filename` on your changes, if you are commiting a .c file?
- [x] Did you run lint -sf `filename` on your changes, if you are commiting a .sh file?
- [x] Does your code compile properly using `make`?
- [x] Did you related issue number in the description?

## Description

<Write a short description of your changes>

## Technical Debt

<Write about any missing things that you are going to do in the future related to this PR>

### CODE REVIEW

File: code/src/test/test__sbi_get_mvendorid/test__sbi_get_mvendorid.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  uint64_t cause = read_csr(scause);
+
+  switch (cause) {
+    default:
+      break;
+  }
+}
+
+/**
+ * @brief Test case example checks for illegal instruction exception
```
Comment (mmhus) [inline code/src/test/test__sbi_get_mvendorid/test__sbi_get_mvendorid.c]: Update the test description to reflect what you are testing

**Comment by 1danish159** [comment]
Updated the test description for sbi spec section 3.5

**Comment by 1danish159** [comment]
Updated

**Review by mmhus — APPROVED** [review]
LGTM.

**Pull request merged**